README.md

```diff
@@ -57,19 +57,15 @@ If either start_time and end_time are invalid or set to None, the simulation wil
 
 # Developer installation
 
-Clone the repository:
-`git clone https://github.com/urbanopt/urbanopt-ditto-reader.git`
-
-Change directories into the repository:
-`cd urbanopt-ditto-reader`
-
-If you are using an ARM chip (Apple Silicon) computer, install this branch of OpenDSSDirect.py:
-`pip install git+https://github.com/dss-extensions/OpenDSSDirect.py@dss_python-0.12.0`
```

Comment:
Do we remember why we were linking to this specific OpenDSS install version? Was this issue resolved in this latest PR?

Reply:
This is to allow opendssdirect.py 0.6.1 to work on ARM chips. It was resolved in 0.7.0. I'll move this change to #48